src/client.py

```diff
@@ -0,0 +1,47 @@
+"""走行体状況監視モジュール.
+
+走行体の状況を把握し続ける.
+@author: aridome222
+@note 参考 https://qiita.com/hoto17296/items/8fcf55cc6cd823a18217
+"""
+import urllib.request
+import time
+
+
+class Client:
+    """走行体がゴールするまでサーバにリクエストを送り続け、ロボットの状況を把握するクラス."""
+
+    def client(self):
```

Comment:
Clientに色々させすぎだと思う。
ロボットの状態を取得するだけの`get_robot_state()`とかにして、camera_system.py側で繰り返す方が、他の処理と合わせやすいと思う。
```suggestion
    def get_robot_state(self):
```